src/components/config/index.ts

```diff
@@ -105,20 +124,13 @@ export const setupConfigFromFile = async (flags: any, watch: boolean) => {
     const fileWatcher = chokidar.watch(path)
     fileWatcher.on('change', async () => {
       const parsed = parseConfig(path, type)
-      await handshakeAndValidate(parsed)
-      updateConfig(parsed)
-    })
-  }
-
-  if (flags['create-config'] && (flags.har || flags.postman)) {
-    const file = flags.output || 'monika.json'
-    writeFile(file, JSON.stringify(parsed), 'utf8', function (err) {
-      if (err) {
-        log.error('An error occured while writing har config to File.')
-        return log.error(err)
+      if (flags['create-config'] && (flags.har || flags.postman)) {
+        writeConfigFile(flags, parsed)
+        return
```

Comment:
i don't this is needed. this block is executed when there's a change in the file. but when `create-config` is specified, monika doesn't run or even watching the file change.

Reply:
removed